poi-ooxml/src/main/java/org/apache/poi/openxml4j/opc/OPCPackage.java

```diff
@@ -273,21 +276,32 @@ public static OPCPackage open(String path, PackageAccess access)
     *             If the specified file doesn't exist, and a parsing error
     *             occur.
     */
-   public static OPCPackage open(File file, PackageAccess access)
-         throws InvalidFormatException {
-       if (file == null) {
-           throw new IllegalArgumentException("'file' must be given");
-       }
-       if (file.exists() && file.isDirectory()) {
+   public static OPCPackage open(File file, PackageAccess access) throws InvalidFormatException {
+       return open(file.toPath(), access);
+   }
+
+    /**
+     * Open a package.
+     * @param path
+     *            The path of the file to open.
+     * @param access
+     *            Type of access to the file that is allowed
+     * @return A PackageBase object, else <b>null</b>.
+     * @throws InvalidFormatException
+     *             If the specified file doesn't exist, and a parsing error
+     *             occur.
+     */
+   public static OPCPackage open(Path path, PackageAccess access) throws InvalidFormatException {
+       if (Files.exists(path) && Files.isDirectory(path)) {
```

Comment:
this check/call will increase other IOPS in the unpacker with the current implementation, unless we swap to the caching filesystem

Reply:
This IOPS cost was already incurred on the former line 281 of `open(File, PackageAccess)` as a check on the `File` object instead of the `Path`. This is fine to use.

```
   public static OPCPackage open(File file, PackageAccess access)
         throws InvalidFormatException {
       if (file == null) {
           throw new IllegalArgumentException("'file' must be given");
       }
       if (file.exists() && file.isDirectory()) {